src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -313,7 +314,7 @@ function EmojiPickerMenu(props) {
             // Enable keyboard movement if tab or enter is pressed or if shift is pressed while the input
             // is not focused, so that the navigation and tab cycling can be done using the keyboard without
             // interfering with the input behaviour.
-            if (keyBoardEvent.key === 'Tab' || keyBoardEvent.key === 'Enter' || (keyBoardEvent.key === 'Shift' && searchInputRef.current && !searchInputRef.current.isFocused())) {
+            if (!ReportUtils.shouldAutoFocusOnKeyPress(keyBoardEvent)) {
```

Comment:
This change caused a regression in EmojiPickerMenu. When filtering the emoji list, it's focusing the emoji each time backspace is hit

https://github.com/Expensify/App/assets/13985840/385cb972-cff0-44f4-b079-2205826c9147